test/unit/specs/components/Personalization/createComponent.spec.js

```diff
@@ -106,7 +106,6 @@ describe("Personalization", () => {
     expect(isAuthoringModeEnabled).toHaveBeenCalled();
     expect(getDecisionScopes).toHaveBeenCalled();
     expect(hasScopes).toHaveBeenCalled();
-    expect(showContainers).toHaveBeenCalled();
```

Comment:
can you add this line back in, but change it to "not.toHaveBeenCalled()". Following Test Driven Design, you want the test to fail before making the change.

Reply:
ok, I have added it